src/wpilib-ws-robot-endpoint.ts

```diff
@@ -142,6 +146,36 @@ export default class WPILibWSRobotEndpoint extends EventEmitter {
         }
     }
 
+    private _handleReadSimDevices(): void {
+        this._robot.getAllSimDevices().forEach(device => {
+            const deviceUpdates: {[key: string]: any} = {};
+
+            const deviceIdent = device.name + (device.channel !== null ? `[${device.channel}]` : "");
+            if (!this._simDevices.has(deviceIdent)) {
```

Comment:
I'm not sure how this could evaluate to true. getAllSimDevices returns a list of devices from _simDevices.

Reply:
Different `_simDevices`. `getAllSimDevices` is a method on the `WPILibWSRobotBase` and stores a list of registered SimDevices on the robot. The `_simDevices` field here stores the last read values, so that we're not constantly pushing data over the network if nothing has changed.

Reply:
I renamed the property in question to `_simDeviceFields` instead, which should make it a little more clear

Reply:
This is easier to understand. Thanks.